app/src/main/java/com/example/caregiver/model/User.java

```diff
@@ -0,0 +1,30 @@
+package com.example.caregiver.model;
+
+public class User {
```

Comment:
this looks great! maybe in the future we can create 'caregivee' and 'caregiver' objects that extend this class and set it up so caregivee's have tasks, rooms, etc. that will require refactoring through most classes so we can do this in Spring haha